draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+        }
+
+        augment "/nw:networks/nw:network/nw:node" +
+                "/nt:termination-point" +
+                "/ns-topo:slo-sle-policy" +
+                "/ns-topo:custom" +
+                "/ns-topo:service-slo-sle-policy" {
+          when "../../../nw:network-types/ns-topo:network-slice" {
+            description "Augment only for Network Slice topology.";
+          }
+          description "Augment node configuration and state.";
+
+          uses ns-topo-slo-sle-policy;
+        }
+
+
+
+Liu, et al.              Expires 13 August 2023                [Page 17]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+        /* network link augments */
+        augment "/nw:networks/nw:network/nt:link" {
+          when "../nw:network-types/ns-topo:network-slice" {
+            description "Augment only for Network Slice topology.";
+          }
+          description "Augment link configuration and state.";
+          uses ietf-nss:service-slo-sle-policy;
+              uses ns-topo-slo-sle-policy;
+        }
+
+        augment "/nw:networks/nw:network/nt:link" +
+                "/ns-topo:slo-sle-policy" +
+                "/ns-topo:custom" +
+                "/ns-topo:service-slo-sle-policy" {
+          when "../../nw:network-types/ns-topo:network-slice" {
+            description "Augment only for Network Slice topology.";
+          }
+          description "Augment link configuration and state.";
+          uses ns-topo-slo-sle-policy;
+        }
+
+        augment "/nw:networks/nw:network/nt:link" +
+                "/ns-topo:slo-sle-policy" +
+                "/ns-topo:custom" +
+                "/ns-topo:service-slo-sle-policy" +
+                "/ns-topo:steering-constraints" {
+          when "../../../nw:network-types/ns-topo:network-slice" {
+            description "Augment only for Network Slice topology.";
+          }
+          description
+                "Augment IETF network slice services to include steering
+             constraints for links within a resource-based transport
+             network slice.";
+          uses ns-topo-steering-constraints;
+        }
+      }
+   <CODE ENDS>
+
+              Figure 4: YANG model for network slice topology
+
+7.  Manageability Considerations
+
+   To ensure the security and controllability of physical resource
+   isolation, slice-based independent operation and management are
+   required to achieve management isolation.  Each optical slice
+   typically requires dedicated accounts, permissions, and resources for
+   independent access and O&M.  This mechanism is to guarantee the
+   information isolation among slice tenants and to avoid resource
+
+
+
+Liu, et al.              Expires 13 August 2023                [Page 18]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+   conflicts.  The access to slice management functions will only be
+   permitted after successful security checks.
```

Comment:
instead of "will only be", maybe "MUST"?

Reply:
Changed to MUST